FuadYoussef/TwistOnTime
# Cleaning up the select notifications screen.

Now notifications are sorted and the screen only shows the default options if no notifications are selected or only "at time of event" is selected

To test:
when creating or editing a timer:

- Have no notifications check and go to the select notifications screen. Then "At time of event" all the other default options and "custom" should appear on the screen. None of which should be checked
- Create a custom notification. Then on the select notifications screen only "At Time of Event", the custom notification, and "Custom" should appear with the custom notification check
- Return to the create timer screen and then go back to the select notification screen to make sure the same notifications are checked/visible
- Change the notifications so that only "At time of event" is selected. Then return to the create timer screen and the go back to the select notifications screen. Now the "At time of event" should be checked, and all the defaults should be there.
- Finally select one of the defaults (for example check "1 minute") and the create a custom notification for a shorter time (like 1 second). After creating the custom notification the select notifications screen should show the notifications in the correct order (at time of event, 1 second before, 1 minute before)
